packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -129,8 +129,11 @@ export const IdvDocSubmitOnSignup = ({
         }
 
         if (!document_number) {
+            const document_name = document_type.text.toLowerCase();
+
             errors.document_number =
```

Comment:
should this be errors.document_name?

Reply:
It's accepting `document_number` in the textfield, I think `errors.document_number` variable's name is fine